### ISSUE TITLE
 v20200710/v20200709 same release?

### ISSUE THREAD

**Issue by danquack** [awslabs/amazon-eks-ami#511]
This month release doesn't seem to have any of the changes from the master branch. When i go over to the tag v20200710, the last commit was back from June. Was the release cut wrong? Is there a reason why the changes after [June 11 commit](https://github.com/awslabs/amazon-eks-ami/commit/5bac3ca1ca6ab0e00dfce3f10c4b2481694609dc) werent added to the July cut?

**Environment**:
- AMI Version: v20200710

```
```

**Comment by heybronson** [comment]
v20200710/v20200709 are essentially the same release. We had to release v20200710 due to an internal issue with release v20200709 (not related to the AMI itself).

**Comment by danquack** [comment]
@heybronson  Thanks for letting me know. Is theres another planned July release with that in mind?

**Comment by heybronson** [comment]
@danquack Yes, we are working on a July release which will contain the latest commits.
